airbyte-integrations/connectors/source-persistiq/source_persistiq/manifest.yaml

```diff
@@ -23,7 +23,7 @@ definitions:
       type: "DefaultPaginator"
       pagination_strategy:
         type: "CursorPagination"
-        cursor_value: "{{ last_records['next_page'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object